easybuild/easyblocks/s/sepp.py

```diff
@@ -0,0 +1,71 @@
+##
+# Copyright 2009-2020 Ghent University
+#
+# This file is part of EasyBuild,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/easybuilders/easybuild
+#
+# EasyBuild is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation v2.
+#
+# EasyBuild is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with EasyBuild.  If not, see <http://www.gnu.org/licenses/>.
+##
+"""
+EasyBuild support for building and installing SEPP, implemented as an easyblock
+@author: Pavel Grochal (INUITS)
+"""
+import os
+
+from easybuild.tools.filetools import copy_dir, write_file, apply_regex_substitutions
+from easybuild.tools.run import run_cmd
+
+from easybuild.easyblocks.generic.pythonpackage import PythonPackage
+
+
+class EB_SEPP(PythonPackage):
+    """Support for installing the SEPP Python package as part of a Python installation."""
+
+    def configure_step(self, *args, **kwargs):
+        """Build sepp using setup.py."""
+        super(EB_SEPP, self).configure_step(*args, **kwargs)
+
+        # Configure sepp
+        run_cmd("python setup.py config -c")
+
+    def install_step(self, *args, **kwargs):
+        super(EB_SEPP, self).install_step(*args, **kwargs)
```

Comment:
also add docstring here please, give short summary of extra install steps custom to SEPP